acropolis-service/src/main/kotlin/org/ephyra/acropolis/service/api/ISystemSoftwareService.kt

```diff
@@ -4,6 +4,5 @@ import org.ephyra.acropolis.persistence.api.entity.SystemSoftwareEntity
 
 interface ISystemSoftwareService {
     fun create(projectId : Long, name: String)
```

Comment:
at some point these will need doc comments on them which is why I left a gap - to cause myself guilt :P